src/api/schema/types/Project.graphql

```diff
@@ -7,8 +7,9 @@ type Project {
 }
 
 type CreateProjectResponse {
-  project: Project
+  status: Status!
   errors: [ValidationError]!
+  project: Project
 }
```

Comment:
I was expecting to be able to write:

```graphql
type CreateProjectResponse implements Response {
  project: Project
}
```

But you have to repeat the `status: Status!` and `errors: [ValidationError]!` everywhere so that’s not so useful.